app/pages/Page.scala

```diff
@@ -16,7 +16,14 @@
 
 package pages
 
-trait Page
+import models.{Mode, UserAnswers}
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+trait Page {
+  def path: JsPath
+  def route(userAnswers: UserAnswers, mode: Mode): Option[Call] = None
```

Comment:
@joecefnhedog because `Section` also extends `Page` we have a lot of "pages" which don't have a route. For instance, any "section" which does not have a corresponding CYA or add-another page will have an undefined route. We have this behaviour across the departures frontends also.

Reply:
Yeah, makes sense to do this then